src/wolfgame.py

```diff
@@ -7354,9 +7377,9 @@ def fgame(cli, nick, chan, rest):
 
             if gamemode not in var.GAME_MODES.keys() - var.DISABLED_GAMEMODES:
                 gamemode = gamemode.split()[0]
-                gamemode, _ = complete_match(gamemode, var.GAME_MODES.keys() - var.DISABLED_GAMEMODES)
+                gamemode = complete_one_match(gamemode, var.GAME_MODES.keys() - var.DISABLED_GAMEMODES)
                 if not gamemode:
-                    cli.notice(nick, messages["invalid_mode_no_list"].format(rest))
+                    cli.notice(nick, messages["invalid_mode_no_list"].format(rest[0]))
```

Comment:
Why are you using `rest[0]` here? It doesn't seem like `rest` is modified, so that would return the first *character*. If you meant the first word, use `rest.split()[0]`.